vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListSpacesController.java

```diff
@@ -33,4 +40,10 @@ public String listSpaces(Model model) {
 
         return "staff/spaces/spacelist";
     }
+    
+    @RequestMapping("/staff/module/slide/listspaces")
+    public ResponseEntity<List<ISpace>> listSpacesForSlideContentBlock(){
+        List<ISpace> spaces = spaceManager.getAllSpaces();
```

Comment:
this still loads all spaces. Instead it should take a query string and then search spaces for the query string (let's just go with searching in the space title for now).